src/assets/v1.3.0/proxy_factory.json

```diff
@@ -42,6 +42,7 @@
     "1287": "0xa6B71E26C5e0845f74c812102Ca7114b6a896AB2",
     "1984": "0xa6B71E26C5e0845f74c812102Ca7114b6a896AB2",
     "2001": "0xa6B71E26C5e0845f74c812102Ca7114b6a896AB2",
+    "2002": "0xE89ce3bcD35bA068A9F9d906896D3d03Ad5C30EC",
```

Comment:
Could you review the addressess? some of them are not the deterministic addresses, this is one case. 
```
reusing "SimulateTxAccessor" at 0x59AD6735bCd8152B84860Cb256dD9e96b85F69Da
deploying "GnosisSafeProxyFactory"
```